traitsui/ui_traits.py

```diff
@@ -43,27 +43,28 @@
     Str,
     Trait,
     TraitError,
-    TraitPrefixList,
     TraitType,
 )
-
+try:
+    from traits.api import PrefixList
+except ImportError:
+    def PrefixList(args, **kwargs):
+        from traits.api import TraitPrefixList
+        return Trait(args[0], TraitPrefixList(*args), **kwargs)
```

Comment:
traitsui master is targeting traitsui version 7.1.0 and i think we will be able to drop support of traits 6 and set minimum version of traits to 6.1.0. Given that, i think we can remove the try/except and simply use `PrefixList`

Reply:
I am not sure about dropping support for traits 6. We should get a confirmation before ditching this compatibility layer.

Reply:
Let's discuss in https://github.com/enthought/traitsui/issues/1054
I think it is a great PR to eliminate the warnings and we should let this in. Maybe we want to keep this compatibility layer for now to decouple this PR with #1054?